src/modules/wizeng/WizengModule.v3

```diff
@@ -150,6 +169,36 @@ class WizengModule extends HostModule("wizeng") {
 		System.write(1, range.result);
 		return HostResult.Value0;
 	}
+	// Exported: output a range of bytes directly to stdout.
+	def puts_array(args: Range<Value>) -> HostResult {
+		match (args[0]) {
+			Ref (object) => {
+				match (object) {
+					null => return HostResult.Throw(Trap.new(TrapReason.NULL_DEREF, null, null));
+					arr: HeapArray => {
+						if (ArrayDecl.!(arr.decl).elem_types[0].pack != Packedness.PACKED_I8) return HostResult.Throw(Trap.new(TrapReason.FAILED_CAST, null, null));
+						match (args[1]) {
```

Comment:
Similarly here, you can just used `Values.unbox_i`.

Reply:
Fixed.